preset/rabbitmq/preset.go

```diff
@@ -22,6 +26,14 @@ const defaultVersion = "alpine"
 const defaultPort = 5672
 const managementPort = 15672
 
+// Message is a single message sent to RabbitMQ.
+type Message struct {
+	Queue       string `json:"queue"`
+	ContentType string `json:"contentType"`
+	StringBody  string `json:"stringBody"`
```

Comment:
I see that json tags still use `camelCase` while all other tags, including the ones in this file, use `snake_case`. Would you mind applying this small change?

Reply:
I'm so sorry. I saw that review but I thought you meant to use camel case (I didn't look at the review in depth 😅). No problem, I'll fix this right away.